src/static/intro-page.css

```diff
@@ -0,0 +1,246 @@
+:root {
+  --color-primary: #0699FF;
+
+  --color-white: #fff;
+  --color-black: #000;
+
+  --font-jost: "Jost", sans-serif;
+  --font-fira-sans: "Fira Sans", serif;
+  --font-roboto: "Roboto", serif;
+}
+
+body {
+  margin: 0;
+  font-family: var(--font-jost);
+  font-weight: 400;
+  letter-spacing: -0.06px;
+}
+
+.container {
+  max-width: 1200px;
+  padding: 0 24px;
+  margin: 0 auto;
+}
+
+.btn-primary {
+  color: var(--color-primary);
+  border: 1px solid var(--color-primary);
+  text-decoration: none;
+  padding: 8px 12px;
+  border-radius: 4px;
+  font-family: var(--font-roboto);
+  transition: all 0.2s ease-in-out;
+}
+
+.btn-primary:hover,
+.btn-primary:focus {
+  color: var(--color-white);
+  background-color: var(--color-primary);
+}
+
+/**
+ * Header.
+ */
+.header-content {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 42px;
+  padding-bottom: 42px;
+  animation: 1.5s fadeInUp;
+}
+
+/**
+ * Menu.
+ */
+.navigation .menu {
+  display: flex;
+  align-items: center;
+  gap: 24px;
+  list-style-type: none;
+  padding: 0;
+  margin: 15px 0;
+}
+
+.navigation .menu a {
+  color: var(--color-black);
+  text-decoration: none;
+  transition: all 0.2s ease-in-out;
+}
+
+.navigation .menu a:hover,
+.navigation .menu a:focus {
+  color: var(--color-primary);
+}
+
+/**
+ * Banner.
+ */
+.banner {
+  text-align: center;
+  padding: 110px 0;
+  animation: 2s fadeInUp;
+}
+
+.banner .title {
+  font-size: 72px;
+  font-weight: 700;
+  margin: 0;
+  font-family: var(--font-fira-sans);
+}
+
+.banner .description {
+  font-size: 24px;
+  line-height: 1.5;
+  margin-top: 16px;
+}
+
+/**
+ * Content.
+ */
+.background-image-section {
+  background-image: url("../assets/images/intro-background.svg");
```

Comment:
These weren't loading for me. So I moved the `images` directory into the `static` directory so that `npm run build` will copy them into `dist`